### DISCUSSION TITLE
Pyansys is lack of one module for dealing with the 3D model.

### DISCUSSION THREAD

**Discussion by dunshan1987** [ansys/pyansys#326] [Q&A] [answered]
When can we see pythonic spaceclaim?

**Comment by RobPasMue** [comment, marked as answer]
Hi @dunshan1987 - stayed tuned for end of September 2023! Ansys will be releasing a PyAnsys package which will allow you to interact with SpaceClaim, Discovery and the Ansys Geometry Service, a new lightweight modeler for performing CAD operations.
